### PR TITLE
pipboard: add show cases to pipboard

### PR DESCRIPTION
**Pull request by utkobe**
This PR includes:
* decouple server part of pipboard from pipcook-core. Before it's using fastify inside pipcook-core. Now I have moved the server into 'pipcook-board-server' and build on top of Egg.js of Alibaba
* add two show cases to the home of pipboard
* optimize pipcook-cli to supoort pipboard and also do some optimization in cli tool
* add 'who's using it' part to Readme and a guide to try MNIST example

Please have a review pls, Thank you @yorkie

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,25 +70,31 @@ Initialize a project:
 ```sh
 $ mkdir pipcook-example && cd pipcook-example
 $ pipcook init
-$ cd pipcook-project
 ```
 
+### Show Cases
+If you are wondering what you can do in pipcook and where you can check your training logs and models, you could run pipboard first.
```
Comment (yorkie) [inline README.md]: Add a blank line please :)

File: README.md [hunk 2]
```diff
@@ -70,25 +70,31 @@ Initialize a project:
 ```sh
 $ mkdir pipcook-example && cd pipcook-example
 $ pipcook init
-$ cd pipcook-project
 ```
 
+### Show Cases
```
Comment (yorkie) [inline README.md]: How about "Playground" or "Pipboard"?

File: README.md (hunk 1 above)
Comment (yorkie) [inline README.md]:
```suggestion
If you are wondering what you can do in [Pipcook][] and where you can check training logs and models, you could start from Pipboard.
```

File: README.md [hunk 3]
```diff
@@ -70,25 +70,31 @@ Initialize a project:
...
+If you are wondering what you can do in pipcook and where you can check your training logs and models, you could run pipboard first.
+```sh
+$ pipcook board
+```
+you will see a web page prompt in your browser. Now you can have a look at MNIST showcase on the home page and play around there. If you want to train a model to recognize MNIST handwritten digits by yourself, you could try the example below.
```
Comment (yorkie) [inline README.md]:
```suggestion
You will see a web page prompt in your browser, and there is a MNIST showcase on the home page and play around there. If you want to train a model to recognize MNIST handwritten digits by yourself, you could try the examples below.
```

File: README.md [hunk 4]
```diff
@@ -70,25 +70,31 @@ Initialize a project:
...
+
 ### Examples
```
Comment (yorkie) [inline README.md]: It should be removed for now, we have mentioned the examples at the previous section.

File: README.md [hunk 5]
```diff
@@ -70,25 +70,31 @@ Initialize a project:
...
-- [pipeline-databinding-image-classification][]: pipeline example to train the iamge classification task which is 
+- [pipeline-mnist-image-classification](example/pipeline-example/pipeline-mnist-image-classification.js): pipeline for classific Mnist image classification problem.
+- [pipeline-databinding-image-classification](example/pipeline-example/pipeline-databinding-image-classification.js): pipeline example to train the iamge classification task which is 
   to classifify [imgcook](https://www.imgcook.com/) databinding pictures.
-- [pipeline-object-detection][]: pipeline example to train object detection task which is for component recognition 
+- [pipeline-object-detection](example/pipeline-example/pipeline-object-detection.js): pipeline example to train object detection task which is for component recognition 
   used by imgcook.
-- [python-keras][]: example to use Python Keras library to train deep leraning network in js syntax and runtime.
+- [python-keras](example/python-nodejs-example/python-keras.js): example to use Python Keras library to train deep leraning network in js syntax and runtime.
```
Comment (yorkie) [inline README.md]: See the bottom of the source: https://raw.githubusercontent.com/alibaba/pipcook/aab7e5d4a7d97166991826ee9966e33806d230e0/README.md, I have defined these links there, is this easier to read?

File: README.md [hunk 6]
```diff
@@ -70,25 +70,31 @@ Initialize a project:
...
 
-See [here](./example) for complete list, and it's easy and quick to run these examples. For example, to do a minst 
+See [here](./example) for complete list, and it's easy and quick to run these examples. For example, to do a MNIST 
 image classification, just run the following to start the pipeline:
 
 ```sh
-$ node examples/pipcook-app-example/pipcook-imageclass-app-test.js
+$ node examples/pipeline-example/pipeline-mnist-image-classification.js
```
Comment (yorkie) [inline README.md]: How about renaming this to examples/pipeline/pipeline-mnist-image-classification?

File: README.md [hunk 7]
```diff
@@ -128,10 +134,13 @@ $ npm run build
 
 ## Community
 
+#### IRC
 <img width="200" src="./community_qrcode.png">
```
Comment (yorkie) [inline README.md]: An empty line is suggested :)

File: README.md [hunk 8]
```diff
@@ -128,10 +134,13 @@ $ npm run build
...
 
 > Download DingTalk (an all-in-one free communication and collaboration platform) here: [English](https://www.dingtalk.com/static/en/download) | [中文](https://page.dingtalk.com/wow/dingtalk/act/download)
 
+#### Who's using it
+<a href="https://www.imgcook.com"><img height="40" src="https://img.alicdn.com/tfs/TB1lle4yQzoK1RjSZFlXXai4VXa-200-64.png"></a>
```
Comment (yorkie) [inline README.md]: Blank lines :)

File: docs/tutorials/get-started-with-pipeline-api.md [hunk 9]
```diff
@@ -145,11 +145,9 @@ const modelEvaluate = ModelEvaluate(classModelEvalute);
 
 Now we have completed every step of the machine learning life cycle, let's pass each plugin to pipcook runner, and tell pipcook to start a runner to begin the training process, as shown below.
 
-```js
-const runner = new PipcookRunner('test1', {
-  predictServer: true
-});
-runner.run([dataCollect, dataAccess, modelLoad, modelTrain, modelEvaluate]);
+```
+const runner = new PipcookRunner()
```
Comment (yorkie) [inline docs/tutorials/get-started-with-pipeline-api.md]:
```suggestion
const runner = new PipcookRunner();
```

File: docs/tutorials/get-started-with-pipeline-api.md [hunk 10]
```diff
@@ -145,11 +145,9 @@ const modelEvaluate = ModelEvaluate(classModelEvalute);
...
+runner.run([dataCollect, dataAccess, modelLoad, modelTrain, modelEvaluate])
```
Comment (yorkie) [inline docs/tutorials/get-started-with-pipeline-api.md]: Ditto.

File: docs/zh-cn/tutorials/get-started-with-pipeline-api.md [hunk 11]
```diff
@@ -131,10 +131,8 @@ const modelEvaluate = ModelEvaluate(classModelEvalute);
 
 现在我们对于机器学习生命周期的每一步的环节就已经编写完成啦，下面我们要将每一个插件传入给 Pipcook runner，并告诉 Pipcook 启动一个 runner 去开始训练流程，如下所示
 
-```ts
-const runner = new PipcookRunner('test1', {
-  predictServer: true
-})
+```
+const runner = new PipcookRunner()
```
Comment (yorkie) [inline docs/zh-cn/tutorials/get-started-with-pipeline-api.md]: Ditto

File: example/pipeline-example/pipeline-mnist-image-classification.js [hunk 12]
```diff
@@ -31,8 +31,8 @@ let imageClassLocalModelDeploy = require('@pipcook/pipcook-plugins-image-class-l
 async function startPipeline() {
   // collect mnist data
   const dataCollect = DataCollect(imageMnistDataCollection, {
-    trainingCount:200,
-    testCount: 200
+    trainingCount:8000,
```
Comment (yorkie) [inline example/pipeline-example/pipeline-mnist-image-classification.js]:
```suggestion
    trainingCount: 8000,
```

File: packages/pipcook-board-server/README.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
+# hackernews-async-ts
```
Comment (yorkie) [inline packages/pipcook-board-server/README.md]: What's this?

File: packages/pipcook-board-server/app/utils/index.ts [hunk 14]
```diff
@@ -0,0 +1,17 @@
+import { Context } from 'egg';
```
Comment (yorkie) [inline packages/pipcook-board-server/app/utils/index.ts]: Just curious that why not using the midway?

File: packages/pipcook-cli/assets/example/pipeline-example/pipeline-mnist-image-classification.js [hunk 15]
```diff
@@ -31,8 +31,8 @@ let imageClassLocalModelDeploy = require('@pipcook/pipcook-plugins-image-class-l
 async function startPipeline() {
   // collect mnist data
   const dataCollect = DataCollect(imageMnistDataCollection, {
-    trainingCount:200,
-    testCount: 200
+    trainingCount:8000,
```
Comment (yorkie) [inline packages/pipcook-cli/assets/example/pipeline-example/pipeline-mnist-image-classification.js]:
```suggestion
    trainingCount: 8000,
```

**Comment by yorkie** [comment]
And the CI should be fixed, too :)

File: README.md (hunk 2 above)
Comment (utkobe) [reply README.md]: I guess playground is more attractive. I would use 'Playground' :)

**Comment by utkobe** [comment]
@yorkie Thank you for review. Have fixed the codes accordingly.

File: README.md (hunk 4 above)
Comment (yorkie) [reply README.md]: @utkobe Hey, the examples section should be included the playground? One is to remove the title, or downgrade the title level to "#### Examples".

**Comment by yorkie** [comment]
> @yorkie Thank you for review. Have fixed the codes accordingly.

See https://github.com/alibaba/pipcook/pull/52/files#r396464928, this might be the last what I have concerned, otherwise SGTM now :)

File: README.md (hunk 4 above)
Comment (utkobe) [reply README.md]: Yes, that makes sense. I just modified it

**Pull request merged**